lib/Lifters/ValueLifter.cpp

```diff
@@ -258,15 +243,12 @@ llvm::Constant *ValueLifterImpl::GetPointer(uint64_t ea, llvm::Type *value_type,
   }
 
   // Wrap the returned pointer in an alias.
-  const auto type = ptr_type->getElementType();
-
   std::stringstream ss;
-  ss << kGlobalAliasNamePrefix << std::hex << ea << '_'
-     << type_specifier.EncodeToString(type,
-                                      EncodingFormat::kValidSymbolCharsOnly);
+  ss << kGlobalAliasNamePrefix << std::hex << ea << '_';
 
   const auto name = ss.str();
-  auto alias_ret = llvm::GlobalAlias::create(value_type, address_space,
+  // TODO(alex): Might need the pointer type here. Try to understand what this is doing later.
```

Comment:
I'm unsure about this spot.

Reply:
I'm going to explore removing this part like we discussed.

Reply:
[fa59b3f](https://github.com/lifting-bits/anvill/pull/296/commits/fa59b3f7fac4f841ea571c3183c0cf2e147a759b)

Not sure if this is really fine. It doesn't cause problems with my `cat` program that I'm testing but I'm more just testing that the lift completes successfully.